parts/k8s/containeraddons/1.11/kubernetesmasteraddons-metrics-server-deployment.yaml

```diff
@@ -126,7 +127,7 @@ spec:
         imagePullPolicy: IfNotPresent
         command:
         - /metrics-server
-        - --source=kubernetes.summary_api:''
+        - --kubelet-insecure-tls
```

Comment:
Shouldn't we apply `--authorization-mode=Webhook` instead?

Reply:
We are doing that:

https://github.com/Azure/aks-engine/blob/master/pkg/api/defaults-kubelet.go#L21

(Or do you mean there's a comparable arg context for metrics-server? Or are you suggesting that in a kubelet context w/ `--authorization-mode=Webhook` we don't need to be configuring metrics-server w/ `--kubelet-insecure-tls` ?)